ibis/udf/vectorized.py

```diff
@@ -18,8 +18,13 @@ class UserDefinedFunction(object):
     def __init__(self, func, func_type, input_type, output_type):
         self.func = func
         self.func_type = func_type
-        self.input_type = input_type
-        self.output_type = output_type
+
+        self.input_type = list(map(dt.dtype, input_type))
+
+        if type(output_type) is list:
+            [output_type] = output_type
```

Comment:
It's more comment to use 

`output_type, = output_type`

Reply:
should probably also catch the exception when the list has more then one element and give a reasonable error message

Reply:
> It's more comment to use
> 
> `output_type, = output_type`

blacken was changing this to `(output_type,) = output_type` so `[output_type] = output_type` seemed cleaner, however, if `output_type,` is more common then maybe `(output_type,)` is more readable


> should probably also catch the exception when the list has more then one element and give a reasonable error message

Will do